DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/AddRegistrationPromptSummary.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+@using Microsoft.Extensions.Configuration
+@model AddRegistrationPromptSummaryViewModel
+
+@{
+  ViewData["Title"] = "Add delegate registration prompt";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["CurrentSystemBaseUrl"]}/tracking/dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-u-margin-bottom-8">
+      <h1 id="page-heading" class="nhsuk-heading-xl">Summary</h1>
+    </div>
+
+    <dl class="nhsuk-summary-list">
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Prompt
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.PromptName
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="AddRegistrationPromptSelectPrompt">

```

Comment:
Ah, looks like you also have the nhsuk-links that are not in official examples :)